src/Joomlatools/Console/Command/Database/Install.php

```diff
@@ -92,25 +92,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $imports = $this->_getSQLFiles($input, $output);
 
-        foreach($imports as $import)
-        {
-            $tmp      = tempnam('/tmp', 'dump');
-            $contents = file_get_contents($import);
-            $contents = str_replace('#__', 'j_', $contents);
-
-            file_put_contents($tmp, $contents);
-
-            $password = empty($this->mysql->password) ? '' : sprintf("-p'%s'", $this->mysql->password);
-            $result = exec(sprintf("mysql --host=%s --port=%u --user='%s' %s %s < %s", $this->mysql->host, $this->mysql->port, $this->mysql->user, $password, $this->target_db, $tmp));
-
-            unlink($tmp);
-
+        foreach ($imports as $import) {
+            $content = file_get_contents($import);
+            if (strlen($content) < 2) {
```

Comment:
it's just a remainder of an earlier solution, where I tried to read the file line-by-line with a generator, I will replace with file_exists()